src/Bicep.Core.UnitTests/Utils/CompilationHelper.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using System;
+using Bicep.Core.Diagnostics;
+using Bicep.Core.SemanticModel;
+using Bicep.Core.UnitTests.Assertions;
+
+namespace Bicep.Core.UnitTests.Utils
+{
+    public class CompilationHelper
```

Comment:
Why do we need this? It's a wrapper for calling a lambda that then calls a diagnostics assertion. Why not just do it imperatively in the code?

Reply:
I put this in before writing the custom fluent assertions, which now make it unnecessary. Will remove it.